README.rst

```diff
@@ -720,6 +727,13 @@ the run step:
         <or>
         platform: linux/arm64/v8 # an apple m1 architecture
 
+        # Multi-platform images
+        # Specify a list of platforms to build for, and buildrunner will build a manifest
+        # list image with the given platforms
+        platform:
+          - linux/amd64
+          - linux/arm64/v8
```

Comment:
Does this make sense here in the `run` section? Not sure what it does with it in this case, doesn't it just add it to the docker pull commands, etc?

Reply:
You are correct! This does not make sense in `run`. I didn't go up enough in the file to make sure this was part of the `build` step.

Reply:
Removed this.